mmengine/config/config.py

```diff
@@ -1029,7 +1029,7 @@ def __call__(self,
             option_string (list[str], optional): Option string.
                 Defaults to None.
         """
-        options = {}
+        options = copy.copy(getattr(namespace, self.dest, None) or {})
```

Comment:
Please add a comment for this modification.

Reply:
Done